crates/orchestrator/src/tests/database/mod.rs

```diff
@@ -1,7 +1,73 @@
+use crate::config::config;
+use crate::jobs::types::{ExternalId, JobItem, JobStatus, JobType};
+use crate::tests::common::{build_config, drop_database};
+use color_eyre::eyre::eyre;
+use dotenvy::dotenv;
 use rstest::*;
+use uuid::Uuid;
 
 #[rstest]
 #[tokio::test]
-async fn test_database() {
-    // TODO: write test case
+async fn test_database_connection() -> color_eyre::Result<()> {
+    let init_config_error = build_config().await.is_err();
+    if init_config_error {
+        return Err(eyre!("Not able to init config."));
+    }
+
+    Ok(())
+}
+
+/// Tests for `create_job` operation in database trait.
+/// Creates 3 jobs and asserts them.
+#[rstest]
+#[tokio::test]
+async fn test_database_create_job() -> color_eyre::Result<()> {
```

Comment:
I don't think this is the correct use of fixture, check this - https://docs.rs/rstest/latest/rstest/attr.fixture.html#async

Reply:
this is still not fixed